docs/se/guides/health.adoc

```diff
@@ -303,6 +352,40 @@ ObserveFeature observe = ObserveFeature.builder()
 curl http://localhost:8080/myhealth
 ----
 
+Earlier you added health config to the `application.yaml` config file to turn on detailed output. If you want to run an experiment, change that `details` setting in the config file to `false` and stop, rebuild, and rerun the application. Now access the health endpoint (at `/myhealth`, remember).
+The output _remains_ detailed because your code--which has full responsibility for determining the custom health observer's behavior--does not apply configuration to the custom observer's builder.
+
+==== Adding configuration to a custom observer
+In addition to preparing the health observer builder with hard-coded settings, your code can also apply configuration for health so your user could set any health behavior there as well.
+
```

Comment:
Was confused by "your user" here. Maybe rephrase this? Maybe "your code can also apply configuration by directly referring to a config node" or something like that.

Reply:
I've rephrased this in a way that, I hope, is clearer.